### ISSUE TITLE
Unable to start server in Windows (Error: failed to initialize logging)

### ISSUE THREAD

**Issue by tripplet** [joseluisq/static-web-server#248]
**Describe the bug**
Static-Web-Server is always failing to start with the error seen below.
Version 2.20.0 works fine, the bug was introduced with 2.20.1

**To Reproduce**
.\static-web-server.exe --port 5555 --root .

**Expected behavior**
Start static-web-server

**Logs/Screenshots**
```
PS C:\> .\static-web-server.exe --port 5555 --root .
Error: failed to initialize logging

Caused by:
    a global default trace dispatcher has already been set
```

**Environment and Specs**
 - static-web-server: v2.20.1
 - OS: Windows

**Comment by joseluisq** [comment]
This is already fixed and should be landed in the next patch release.
Thanks for reporting!

**Comment by joseluisq** [comment]
Resolved on [v2.20.2](https://github.com/static-web-server/static-web-server/releases/tag/v2.20.2)

**Comment by tripplet** [comment]
Thanks for fixing, works now
